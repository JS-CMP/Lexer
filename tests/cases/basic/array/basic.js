const array = [
    undefined,
    null,
    true,
    false,
    0,
    1,
    1.1,
    'a',
    '',
    "a",
    ""
]
