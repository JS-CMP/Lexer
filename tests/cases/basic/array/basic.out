#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"
#include "customOperators/CustomOperators.hpp"

int main() {JS::Any array;array=JS::Any(JS::Array({JS::Any(), JS::Any(nullptr), JS::Any(true), JS::Any(false), JS::Any(0), JS::Any(1), JS::Any(1.1), JS::Any("a"), JS::Any(""), JS::Any("a"), JS::Any("")}));}
